### DISCUSSION TITLE
0.2.0

### DISCUSSION THREAD

**Discussion by MasterJayanX** [MasterJayanX/pmfow#7] [Announcements]
## Changelog
- The extension for each package is now determined by the URL of the file.
- Switched from .txt to .dat files for the repositories.
- Added a new command: `uninstall`, which allows you to uninstall packages that you have.
- Added a check for new updates.
- Deprecated Powershell as a way to download files.
- New flags: `--unstable`, `--uninstall`, `--check`: `--unstable` allows the update command to check for unstable/development releases. `--uninstall` lists the programs that can be uninstalled. `--check` will only check for updates instead of updating the repositories as well.
- The version command now works on Windows 9x (keep in mind that other commands are still restricted to Windows 2000 and later versions of Windows).

## Installation
Download the file named pmfow_win32.exe or pmfow_win64.exe and install it. The other 2 files (pmfow32.exe and pmfow64.exe) are just the standalone pmfow executables for 32 and 64 bit respectively, but without the necessary dependencies (these are used by the pmfow-updater tool).

<hr /><em>This discussion was created from the release <a href='https://github.com/MasterJayanX/pmfow/releases/tag/0.2.0'>0.2.0</a>.</em>

**Comment by nixxoq** [comment]
Hello, what about adding some applications from my repository [xp-apps](https://github.com/snaky1a/xp-apps)?

**Reply by MasterJayanX** [reply]
I might consider it. Thanks for the suggestion.
